common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -5099,9 +5099,15 @@ schematic = (function() {
 	}
 
 	ConnectionPoint.prototype.toString = function() {
-	    return '<ConnectionPoint ('+this.offset_x+','+this.offset_y+') '+this.parent.toString()+'>';
+	    return edx.StringUtils.interpolate('<ConnectionPoint ({offset_x}, {offset_y}) {parent}>',
```

Comment:
FYI: Here and below you are adding a space between x and y that wasn’t there, if it matters.